ui/Screen/Lock.svelte

```diff
@@ -5,11 +5,14 @@
 
   let passphrase = "";
 
-  const unlock = () => {
-    session.unseal(passphrase);
+  const unlock = async () => {
+    const unlocked = await session.unseal(passphrase);
+    if (!unlocked) {
+      passphrase = "";
+    }
   };
 
-  const onEnter = () => {
+  const onEnter = async () => {
```

Comment:
Does this really need to be `async` if you don't `await` in the body?

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1170/commits/b0470946bc21f7a0d76e00d02cab481f60ef86b1